src/fosslight_util/write_excel.py

```diff
@@ -82,18 +82,29 @@ def remove_empty_sheet(sheet_items):
     return success, final_sheet_to_print
 
 
-def get_header_row(sheet_name, sheet_content):
+def get_header_row(sheet_name, sheet_content, extended_header):
     selected_header = []
+
+    if extended_header:
```

Comment:
Even if `extended_header ` is` {}`, `if extended_header `will be True.
Let's remove `if extended_header `.